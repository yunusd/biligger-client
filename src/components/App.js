import React from 'react';
import { ApolloProvider } from 'react-apollo';

import { Container } from 'semantic-ui-react';

import './App.css';

import client from './ApolloClient';

import Header from './Header';
import Feed from './Feed';

const App = () => (
  <ApolloProvider client={client}>
    <Container fluid>
      <Header />
      <Feed />
    </Container>
  </ApolloProvider>
);

export default App;
