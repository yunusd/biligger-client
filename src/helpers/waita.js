export default (amount = 0) => new Promise(resolve => setTimeout(resolve, amount));
