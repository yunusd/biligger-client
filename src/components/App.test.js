import React from 'react';
import ReactDOM from 'react-dom';

import { mount } from 'enzyme';
import { BrowserRouter as Route } from 'react-router-dom';

import waita from '../helpers/waita';

import App from './App';


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Route><App /></Route>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', async () => {
  const app = mount(<Route><App /></Route>).html();
  await waita(0);
  expect(app).toMatchSnapshot();
});
