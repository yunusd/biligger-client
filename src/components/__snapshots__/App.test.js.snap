// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"ui fluid container\\"><div><div class=\\"ui secondary menu\\"><div class=\\"left header item\\"><a href=\\"/\\"><img src=\\"logo.png\\" width=\\"100px\\" class=\\"ui image\\"></a></div><div class=\\"item header-right\\"><button class=\\"ui basic button\\">Giriş Yap</button></div><div class=\\"item header-right\\">yada</div><div class=\\"item header-right\\"><button class=\\"ui green basic button\\">Kayıt Ol</button></div></div><div class=\\"ui divider\\"></div></div><div class=\\"ui active transition visible inverted dimmer\\" style=\\"display: flex;\\"><div class=\\"content\\"><div class=\\"ui inverted text loader\\">Yükleniyor</div></div></div></div>"`;
